app/views/visitor_mailer/booked.html.erb

```diff
@@ -32,6 +32,20 @@
   <p><%= t(".this_is_a_closed_visit_html") %></p>
 <% end %>
 
+<% if @visit.visitors.any?(&:banned?) %>
```

Comment:
Does line [19](https://github.com/ministryofjustice/prison-visits-2/pull/166/files#diff-0613bf27f26accef00d1ecc1a3b22690R19) need to be updated to exclude the banned visitors?